docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -0,0 +1,458 @@
+***********************
+Using Sawtooth with SGX
```

Comment:
Consider titling this around PoET. Maybe 'Using Sawtooth with the SGX implementation of PoET'